eclair-core/src/main/scala/fr/acinq/eclair/channel/ChannelFeatures.scala

```diff
@@ -92,23 +94,35 @@ sealed trait SupportedChannelType extends ChannelType {
 object ChannelTypes {
 
   // @formatter:off
-  case object Standard extends SupportedChannelType {
-    override def features: Set[ChannelTypeFeature] = Set.empty
+  case class Standard(scidAlias: Boolean, zeroConf: Boolean) extends SupportedChannelType {
```

Comment:
Default values would make sense here I think, and would make tests a bit less verbose.
```suggestion
  case class Standard(scidAlias: Boolean = false, zeroConf: Boolean = false) extends SupportedChannelType {
```

Reply:
Done in https://github.com/ACINQ/eclair/pull/2404/commits/6979d3f8536adfe89979bf1049f3884665cfa69b

Reply:
This could be done for the other classes as well don't you think? Sorry if that wasn't clear.

Reply:
I hesitated, since you specifically highlighted this one I think you only wanted it. In that case we should do it for all of them, including anchor-outputs-zero-fees, right?

Reply:
Yes I guess so.

Reply:
Done in https://github.com/ACINQ/eclair/pull/2404/commits/96d7a478987a06c048a20040f4a4a2775a49fd76, this feels lighter!